cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js

```diff
@@ -29,7 +29,7 @@ describe('ui-data-import', () => {
   const collectionOfMappingAndActionProfiles = [
     {
       mappingProfile: { name: `C380446 Test P/E mix open order with instance, holdings, item ${getRandomPostfix()}`,
-        typeValue : NewFieldMappingProfile.folioRecordTypeValue.order,
+        typeValue: FOLIO_RECORD_TYPE.ORDER,
         orderStatus: 'Open',
```

Comment:
this value of orderStatus can be moved to constant

Reply:
Done